other-docs/guides/how-to-submit-a-bug-report.md

```diff
@@ -0,0 +1,19 @@
+# How to submit a bug report.
+
+Bugs happen, and we’re here to support you, should you find a bug in Altis.
+
+To get the most out of support and to help our product team, there are some requirements for a bug report that must be met, in order for us to help you with your issue.
+
+When filing a bug report ( [you can read this guide on how to file support tickets](https://docs.altis-dxp.com/nightly/guides/getting-help-with-altis/) ) please provide all of the following:
+## Test Results.
+
+Please write a brief description of the bug, including what you expect to happen and what is currently happening.
+## [Reduced Test Case](https://css-tricks.com/reduced-test-cases/).
+
+Please ensure you test the bug in as a reduced test case, and provide the reduced test case as the basis of the bug report.
+## Steps to reproduce.
+
+Detailed step-by-step instructions to reproduce the issue.
+## Environment Setup and Configuration.
```

Comment:
```suggestion
If possible, please also provide detailed step-by-step instructions to reproduce the issue. This will ensure that we can replicate the problem, and avoids back-and-forth conversations.

## Environment Setup and Configuration.
```